buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -172,7 +172,7 @@ class BuildPlugin implements Plugin<Project>  {
         project.repositories.mavenCentral()
         project.repositories.maven { url "https://conjars.org/repo" }
         project.repositories.maven { url "https://clojars.org/repo" }
-        project.repositories.maven { url 'https://repo.spring.io/plugins-release' }
+        project.repositories.maven { url 'https://repo.spring.io/plugins-release-local' }
```

Comment:
Why was this required _in addition_ to dropping the dependency?

Reply:
I've reverted this. It doesn't seem like we actually make use of this repository at all, especially since as it is it requires authorization to access. I'll put out a follow up to clean it out.